ms18-applicatie-console/DatabaseTestData.cs

```diff
@@ -75,7 +74,7 @@ private void CreateTestDataMember()
                 {
                     ["Admin"] = admin,
                  
-                    // Team A
+                    // Team B
                     ["Gast"] = new Member() { Name = "Gast", MemberCreated = admin, Email = "gast@example.com", Password = verySafePassword },
```

Comment:
Gastgebruikers hoeven geen wachtwoord, maar als ik dit zo zie lijkt dit nu wel het geval te zijn?

Reply:
Je hebt gelijk, dat had ik al eens veranderd: De endpoints voor het bekijken van producten en het plaatsen van een bestelling zijn al bruikbaar zonder in te loggen.

Maar in de testdata had ik inderdaad een "Gast" account, dat is ondertussen een standaard scoutinglid geworden (die bestellingen mag bekijken en declaraties mag indienen), dus de naam "Gast" is inderdaad niet zo handig. Ik heb er nu "Lid" van gemaakt.